recipes/cherry-picker/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "cherry-picker" %}
```

Comment:
Please remove all of the jinja variables except `name` and `version` and then just put them in explicitly in the recipe.

Reply:
This usage of named placeholders is standard across the numerous feedstocks we maintain (and at least `sha256` widely used in other feedstocks), single-sourcing the common variables in one place rather than repeating them scattered throughout the business logic, and makes workflows more resuable, syncable and consistent between feedstocks, decreasing overall maintainer burden while increasing quality. Furthermore, it makes _more_ clear and explicit which of the several similar or identical names each value actually represents, and allows trivially changing any of them while avoiding mistakes.

Given that, is there an established rationale for why `name` and `version` are allowed to be variables, but not `import`, `entrypoint` or `sha256` in the packages we maintain?